src/crt/sllshrs.src

```diff
@@ -0,0 +1,49 @@
+; Performs signed 48 bit ("short long long") bit shift right
+; 
+; Arguments:
+; ude = Most significant bytes of arg 1
+; uhl = Least significant bytes of arg 1
+; ubc = Amount to shift by (effectively just shift by c)
+
+
+; Returns:
+; ude:uhl = ude:uhl>>c
+
+	assume	adl=1
+
+	section	.text
+	public	__sllshrs
+__sllshrs:
```

Comment:
This needs an early return if shifting by 0, otherwise it incorrectly shifts by 256.